bifrost/release.go

```diff
@@ -289,8 +278,6 @@ func (r *Release) CheckUserLock(s3c aws.S3API, lockPath string) error {
 	return nil
 }
 
-// DEPRECATED: this will gradually be replaced by `grabGenericLock` (below),
-// but it is currently preserved for backwards compatibility.
 func (r *Release) grabS3Lock(s3c aws.S3API, lockPath string) error {
```

Comment:
Still want to keep?

Reply:
It's being used by the release lock, so the deprecation notice I put up wasn't quite right hah.